### PR TITLE
Fix some code smells + use editorconfig new rules

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/DoNotUseIIfCodeFixProvider.cs [hunk 1]
```diff
@@ -44,30 +44,30 @@ public override FixAllProvider GetFixAllProvider()
...
 
-        protected override async Task RegisterCodeFixesAsync(SyntaxNode root, CodeFixContext context)
+        protected override Task RegisterCodeFixesAsync(SyntaxNode root, CodeFixContext context)
         {
             var diagnostic = context.Diagnostics.First();
             var diagnosticSpan = diagnostic.Location.SourceSpan;
 
             var iifInvocation = root.FindNode(diagnosticSpan) as InvocationExpressionSyntax;
-            if (iifInvocation == null)
+            if (iifInvocation != null)
```
Comment (michalb-sonar) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Rules/DoNotUseIIfCodeFixProvider.cs]: Not sure that's better, but meh.

File: .editorconfig [hunk 2]
```diff
@@ -53,3 +53,46 @@ dotnet_style_qualification_for_field = false:suggestion
 dotnet_style_qualification_for_property = false:suggestion
 dotnet_style_qualification_for_method = false:suggestion
 dotnet_style_qualification_for_event = false:suggestion
+
```
Comment (michalb-sonar) [inline .editorconfig]: Please split it into a separate commit.

Comment (Evangelink) [reply .editorconfig]: Done
